roles/installer/tasks/install.yml

```diff
@@ -10,9 +10,28 @@
       metadata:
         name: '{{ ansible_operator_meta.name }}'
         namespace: '{{ ansible_operator_meta.namespace }}'
-        labels: '{{ lookup("template", "labels/common.yaml.j2") | from_yaml }}'
+        labels: '{{ lookup("template", "roles/common/templates/labels/common.yaml.j2") | from_yaml }}'
```

Comment:
It looks like CI is failing because it can't find this template.


```
[installer : Start installation] ******************************************\\r\\ntask path: /opt/ansible/roles/installer/tasks/main.yml:11\\nincluded: /opt/ansible/roles/installer/tasks/install.yml for localhost\\n\\r\\nTASK [installer : Patching labels to AWX kind] *********************************\\r\\ntask path: /opt/ansible/roles/installer/tasks/install.yml:2\\n[WARNING]: Unable to find 'roles/common/templates/labels/common.yaml.j2' in\\r\\nexpected paths (use -vvvvv to see paths)\\nfatal: [localhost]: FAILED! => {\\\"msg\\\": \\\"An unhandled exception occurred while running the lookup plugin 'template'. Error was a <class 'ansible.errors.AnsibleError'>, original message: the template file roles/common/templates/labels/common.yaml.j2 could not be found for the lookup\\\"}
```

You may have to use the relative path here.